### ISSUE TITLE
sudo: osxcollector.py: command not found

### ISSUE THREAD

**Issue by tadwhitaker** [Yelp/osxcollector#139]
I copied the osxcollector.py down to my local machine, ran sudo osxcollector.py, was prompted for my password and then received the following error:

sudo: osxcollector.py: command not found

Any suggestions? I also tried cloning the whole repo down and that required pip installing several package dependencies, which the READ.ME file says aren't needed to actually run the check.

**Comment by jjsendor** [comment]
Hi @tadwhitaker,
`osxcollector.py` is in `osxcollector/` directory.

So if you just cloned the GitHub repo, you should go inside the repository and invoke:
`sudo osxcollector/osxcollector.py`

Let me know if that helps!

Also, I will update the [README](https://github.com/Yelp/osxcollector/blob/master/README.md) file as it is quite confusing because it does take into account that `osxcollector.py` is not in the root directory of the file.

**Comment by tadwhitaker** [comment]
Thanks for such a fast response. Just did that and, after being prompted for Password, got the following error:

Traceback (most recent call last):
  File "osxcollector/osxcollector.py", line 41, in <module>
    import macholib.MachO
ImportError: No module named macholib.MachO

The macholib is already present in External Libraries. However, I don’t see macholob.MachO there. Not sure why. Am going to look this up too, but thought I would bring it to your attention as well. Maybe you 

...


Tad Whitaker
twitter.com/theporkskewer

**Comment by jjsendor** [comment]
@tadwhitaker could you look at #137? It seems some other people are reporting similar problems.
tl;dr: I think this is brew messing up with the Python version you are using.

**Comment by tadwhitaker** [comment]
Yeah, I'm having the exact same issue Paula is having. I ran: 

sudo /usr/local/bin/python osxcollector/osxcollector.py

and still got the following error:

Traceback (most recent call last):
  File "osxcollector/osxcollector.py", line 41, in <module>
    import macholib.MachO
ImportError: No module named macholib.MachO

I ran **which python** and it returned the following:

/usr/local/bin/python

**Comment by jjsendor** [comment]
Since this is exactly the same issue as #137 I am closing this one.

@tadwhitaker please look at my last comment there and raise any related questions there if it is still not working for you.
